.github/workflows/frontend-ci.yml

```
@@ -95,26 +97,29 @@ jobs:
+      working-directory: ./frontend-react
       run: npm ci
 
     - name: Run security audit
-      working-directory: ./frontend
+      working-directory: ./frontend-react
       run: npm audit --audit-level=moderate
+      continue-on-error: true
 
     - name: Run Snyk security scan
       uses: snyk/actions/node@v1
+      working-directory: ./frontend-react
```

actionlint (1.7.8)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)